engine/server/engine/main.go

```diff
@@ -458,13 +456,13 @@ func restApiServer(
 	logrus.Infof("Setting-up CORS policy to accept requests from origins: %v", allowOrigins)
 
 	// nolint:exhaustruct
-	echoApiRouter.Use(middleware.CORSWithConfig(middleware.CORSConfig{
+	echoApiRouter.Use(echomiddleware.CORSWithConfig(echomiddleware.CORSConfig{
 		AllowOrigins: allowOrigins,
 		AllowHeaders: defaultCORSHeaders,
 	}))
 
 	// ============================== Engine Management API ======================================
-	engineRuntime := restApi.EngineRuntime{
+	engineRuntime := server.EngineRuntime{
```

Comment:
Tough name.. I almost prefer the old one!

Reply:
agreed - but golangci wouldn't allow two imports of the same path with different names (`restApi` and `server` were coming from the same package). So I went with the broader scoped naming.